TravisCI build broken

seems like the Travis CI build is currently broken?

https://travis-ci.com/etcd-io/jetcd/builds/99918995, for #493, but unrelated; that couldn't cause this IMHO: 

```[ERROR] Tests run: 97, Failures: 1, Errors: 0, Skipped: 1, Time elapsed: 98.234 s <<< FAILURE! - in TestSuite
[ERROR] null on io.etcd.jetcd.WatchResumeTest(io.etcd.jetcd.WatchResumeTest)  Time elapsed: 4.625 s  <<< FAILURE!
java.io.UncheckedIOException: java.nio.file.AccessDeniedException: /tmp/jetcd_test_etcd0_62612002645923743/member
Caused by: java.nio.file.AccessDeniedException: /tmp/jetcd_test_etcd0_62612002645923743/member
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   WatchResumeTest » UncheckedIO java.nio.file.AccessDeniedException: /tmp/jetcd_...
[INFO] 
[ERROR] Tests run: 96, Failures: 1, Errors: 0, Skipped: 0```

Not sure what this is about (it currently passes for me locally) - anyone got a clue?

#491 still passed before yesterday, so just a temporary glitch at Travis?

I by chance just hit this locally also, and dug a bit into it...

so in `jetcd/jetcd-core/target/surefire-reports/testng-junit-results/emailable-report.html` (and also `jetcd/jetcd-core/target/surefire-reports/testng-junit-results/Surefire suite/Surefire test.html`) we find the following, which we couldn't see on Travis:

```
java.io.UncheckedIOException: java.nio.file.AccessDeniedException: /tmp/jetcd_test_etcd0_210268759814789344/member
	at java.nio.file.FileTreeIterator.fetchNextIfNeeded(FileTreeIterator.java:88)
	at java.nio.file.FileTreeIterator.hasNext(FileTreeIterator.java:104)
	at java.util.Iterator.forEachRemaining(Iterator.java:115)
	at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:481)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:471)
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:418)
	at io.etcd.jetcd.launcher.EtcdContainer.deleteDataDirectory(EtcdContainer.java:263)
	at io.etcd.jetcd.launcher.EtcdContainer.close(EtcdContainer.java:174)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at io.etcd.jetcd.launcher.EtcdClusterFactory$2.close(EtcdClusterFactory.java:71)
	at io.etcd.jetcd.launcher.junit.EtcdClusterResource.after(EtcdClusterResource.java:53)
	at org.junit.rules.ExternalResource$1.evaluate(ExternalResource.java:50)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:115)
Caused by: java.nio.file.AccessDeniedException: /tmp/jetcd_test_etcd0_210268759814789344/member
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newDirectoryStream(UnixFileSystemProvider.java:427)
	at java.nio.file.Files.newDirectoryStream(Files.java:457)
	at java.nio.file.FileTreeWalker.visit(FileTreeWalker.java:300)
	at java.nio.file.FileTreeWalker.next(FileTreeWalker.java:372)
	at java.nio.file.FileTreeIterator.fetchNextIfNeeded(FileTreeIterator.java:84)
	... 43 more
```

This should let me fix this - I'll try raise a PR ..

--- a/jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/junit/EtcdClusterResource.java
+++ b/jetcd-launcher/src/main/java/io/etcd/jetcd/launcher/junit/EtcdClusterResource.java
@@ -19,8 +19,13 @@ package io.etcd.jetcd.launcher.junit;
 import io.etcd.jetcd.launcher.EtcdCluster;
 import io.etcd.jetcd.launcher.EtcdClusterFactory;
 import org.junit.rules.ExternalResource;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class EtcdClusterResource extends ExternalResource {
+
+  private static final Logger LOG = LoggerFactory.getLogger(EtcdClusterResource.class);
+
   private final EtcdCluster cluster;
 
   public EtcdClusterResource(String clusterName) {
@@ -50,6 +55,10 @@ public class EtcdClusterResource extends ExternalResource {
 
   @Override
   protected void after() {
-    this.cluster.close();
+    try {
+      this.cluster.close();
+    } catch (RuntimeException e) {
+      LOG.warn("close() failed (but ignoring it)", e);
+    }
   }
 }

